xml/admin_ceph_upgrade.xml

```diff
@@ -1109,6 +1109,28 @@ mds standby for name = mds.456-another-mds
     <link xlink:href="https://documentation.suse.com/suse-distribution-migration-system/1.0/single-html/distribution-migration-system/"/>
     to find both general and detailed information about DMS.
    </para>
+   <important>
+    <para>
+     Before starting the upgrade process, ensure the following packages are not installed:
+    </para>
+    <itemizedlist>
+     <listitem>
+      <para>
+       <package>sles-ltss-release</package>
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       <package>sles-ltss-release-POOL</package>
+      </para>
+     </listitem>
+    </itemizedlist>
+    <para>
+     If the above packages are installed, you can uninstall them with the following command
+     before proceeding:
+    </para>
+<screen>&prompt.sminion;zypper rm -y sles-ltss-release sles-ltss-release-POOL</screen>
+   </important>
```

Comment:
The only issue I have with this addition is we're saying "ensure these packages are installed but if they are, uninstall them" without a lot of context. Personally, I'd prefer some contextual understanding here? 
That, or perhaps we include it as part of the instruction set and add new steps:
1. Check if these packages are installed <command>
2. Optional: uninstall these packages <command>

Reply:
Before the starting the upgrade process, check whether the `sles-ltss-release` and/or `sles-ltss-release-POOL` packages are installed:

```
# rpm -q sles-ltss-release
# rpm -q sles-ltss-release-POOL
```

If either or both are installed, remove them:

```
# zypper rm -y PACKAGE
```

Reply:
I'll take that as an agreement to my second suggestion ;) I like this much better.

Reply:
(added two important caveats to the originally proposed text)